redwood-widget/src/commonMain/kotlin/app/cash/redwood/widget/RedwoodView.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2023 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.redwood.widget
+
+import app.cash.redwood.ui.UiConfiguration
+import kotlin.native.ObjCName
+import kotlinx.coroutines.flow.StateFlow
+
+@ObjCName("RedwoodView", exact = true)
+public interface RedwoodView {
+  public val children: Widget.Children<*>
```

Comment:
This type should probably have a generic which is then used for `children`. Future usages of this type will use the generic to ensure you are using a correctly-typed `Widget.Factory<T>`.

Reply:
Fixed in https://github.com/cashapp/redwood/pull/1442.